RELEASE.md

```diff
@@ -235,7 +235,7 @@ and then "Retry", because after waiting for previous step, artifacts are now rea
     ```bash
     git checkout $BRANCH
     git pull upstream $BRANCH
-    cd backend/api/python_http_client
+    cd backend/api/v2beta1/python_http_client
```

Comment:
Should we add a comment describing how to update `v1beta1` api package?

Reply:
Likely v1 API becomes stable at this point, we should rarely need to release v1 api package. 
In fact, if we do, we might need to do it from a old release branch.

Reply:
That makes sense. Thanks!